k8s/sandbox/common/idam/idam-api.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: idam-api
+  namespace: idam
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: idam-api
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: idam-api
+    version: 0.1.0
+  values:
+    java:
+      image: 'hmctspublic.azurecr.io/idam/api:preview'
+      replicas: 2
+      environment:
+        TESTING_SUPPORT_ENABLED: true
+        IDAM_SPI_FORGEROCK_AM_ROOT: https://forgerock-am.service.core-compute-idam-saat.internal:8443/openam
+        IDAM_SPI_FORGEROCK_AM_TOPLEVELHOST: forgerock-am.service.core-compute-idam-saat.internal:8443
+        IDAM_SPI_FORGEROCK_AM_JWKSURIFOROAUTH2CLIENTS: http://forgerock-am.service.core-compute-idam-saat.internal:8080/openam/oauth2/hmcts/connect/jwk_uri
+        IDAM_SPI_FORGEROCK_IDM_ROOT: https://forgerock-idm.service.core-compute-idam-saat.internal:18443/openidm
+        IDAMHEALTHCHECK_AM_TARGET: http://forgerock-am.service.core-compute-idam-saat.internal:9292
+        IDAMHEALTHCHECK_IDM_TARGET: http://forgerock-idm.service.core-compute-idam-saat.internal:9292
+        SSL_TRUST_DNS_0: C=UK,ST=London,L=London,O=Amido,CN=*.service.core-compute-idam-saat.internal,emailAddress=devops@amido.com
+        SSL_TRUST_DNS_1: C=None,ST=None,L=None,OU=None,O=OpenDJ Self-Signed Certificate,CN=openidm-localhost
+        SSL_TRUST_HOSTNAMES_0: forgerock-idm.service.core-compute-idam-saat.internal
```

Comment:
saat or sandbox?

Reply:
Saat for now. Sandbox = FR 6.5. Saat = FR 5.5.